src/examples/typography/Lists.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { OrderedList, UnorderedList } from '@zendeskgarden/react-typography';
+
+const marginBottom = `margin-bottom: 18px;`;
+
+const Example = () => (
+  <>
+    <OrderedList size="small" css={marginBottom}>
+      <OrderedList.Item>
+        The world&apos;s tallest-growing tree is the coast redwood
+      </OrderedList.Item>
+      <OrderedList.Item>Bamboo can grow 35 inches in a single day</OrderedList.Item>
+      <OrderedList.Item>During the 1600s, tulips were worth more than gold</OrderedList.Item>
+    </OrderedList>
+    <OrderedList size="medium" css={marginBottom}>
+      <OrderedList.Item>
+        The world&apos;s tallest-growing tree is the coast redwood
+      </OrderedList.Item>
+      <OrderedList.Item>Bamboo can grow 35 inches in a single day</OrderedList.Item>
+      <OrderedList.Item>During the 1600s, tulips were worth more than gold</OrderedList.Item>
+    </OrderedList>
+    <UnorderedList size="large">
+      <UnorderedList.Item>
+        The world&apos;s tallest-growing tree is the coast redwood
+      </UnorderedList.Item>
+      <UnorderedList.Item>Bamboo can grow 35 inches in a single day</UnorderedList.Item>
+      <UnorderedList.Item>During the 1600s, tulips were worth more than gold</UnorderedList.Item>
+    </UnorderedList>
```

Comment:
I believe our default is a filled `disc`, not the empty circle. cc @steue check me here.

Reply:
@allisonacs you can see from https://5efe4352c3bae3abf8f9be55--zendeskgarden.netlify.app/components/typography#unorderedlist that we decided to default to `circle`. And I couldn't tell you why, because that doesn't even go with [the grain of the web](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type#Values) 😬. We can decide whether to change this in a future, breaking version 😭.

Reply:
Um… is that true though? Image below taken from current website. Default appears to be `disc`. 
![image](https://user-images.githubusercontent.com/1230973/86815905-0ef72180-c038-11ea-8020-a37ad1e34d43.png)

In this screenshot, `circle` only enters into the picture on nested items. 
![image](https://user-images.githubusercontent.com/1230973/86816066-3fd75680-c038-11ea-85a9-ebbdf95f3340.png)

Reply:
Per internal conversation, we've determined that `circle` as a default setting is a bug, which will be fixed in a future release. We can resolve this comment.